lua/termcolors.lua

```diff
@@ -77,27 +79,32 @@ function termcolors.scrape_current_colorscheme()
 	colors.inactive_border_color = color_from_syntax("VertSplit")
 	colors.bell_border_color = color_from_syntax("TextWarning")
 
-	local cleaned_colors = ordered_table({})
-	for setting_name, setting_value in ordered_pairs(colors) do
+	local cleaned_colors = helpers.ordered_table({})
+	for setting_name, setting_value in helpers.ordered_pairs(colors) do
 		if setting_value ~= nil then
 			cleaned_colors[setting_name] = string.lower(setting_value)
 		end
 	end
 	return cleaned_colors
 end
 
-function termcolors.generate_kitty_config()
-	local config = { "# Put the following lines in your ~/.config/kitty/kitty.conf" }
-	for setting_name, setting_value in ordered_pairs(termcolors.scrape_current_colorscheme()) do
-		table.insert(config, setting_name .. " " .. setting_value)
-	end
-	return config
-end
+function termcolors.show(plugin_name)
+    if plugin_name == nil or plugin_name == "" then
```

Comment:
You seem to have mixed space-based indentation into this tab-indented file. Could you please make it tab-indented everywhere (including newly introduced files)?